Task 0/2019201408陈志朋/hello.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+int main ()
+{
+	printf ("Hello World!\n");
```

Comment:
The space after `printf` is extra. Only add space after `if`, `switch`, `while` and `for`.